flink-libraries/flink-cep/src/main/java/org/apache/flink/cep/nfa/NFA.java

```diff
@@ -19,12 +19,16 @@
 package org.apache.flink.cep.nfa;
 
 import com.google.common.collect.LinkedHashMultimap;
+import org.apache.commons.io.output.ByteArrayOutputStream;
```

Comment:
Why do we use once the `org.apache.commons.io.output.BAOS` and then the `java.io.BAIS`?